packaging/language/pip.py

```diff
@@ -264,14 +277,20 @@ def main():
             if os.path.basename(virtualenv) == virtualenv:
                 virtualenv = module.get_bin_path(virtualenv_command, True)
 
+            cmd = [virtualenv]
             if module.params['virtualenv_site_packages']:
-                cmd = '%s --system-site-packages %s' % (virtualenv, env)
+                cmd.append('--system-site-packages')
             else:
                 cmd_opts = _get_cmd_options(module, virtualenv)
                 if '--no-site-packages' in cmd_opts:
-                    cmd = '%s --no-site-packages %s' % (virtualenv, env)
-                else:
-                    cmd = '%s %s' % (virtualenv, env)
+                    cmd.append('--no-site-packages')
+            python_bin = module.params['python_bin']
+            if python_bin:
+                cmd.append("--python=%s" % python_bin)
+
+            cmd.append(env)
+            cmd = ' '.join(cmd)
```

Comment:
You can pass a list to [module.run_command()](https://github.com/ansible/ansible/blob/5ba3452b7ee0e2abdf61d528e385057faff9e584/lib/ansible/module_utils/basic.py#L1504), so this join is not necessary.